test/check-machines-lifecycle

```diff
@@ -367,6 +367,27 @@ class TestMachinesLifecycle(machineslib.VirtualMachinesCase):
         b.wait_text("h2.vm-name", "test%")
         b.wait_not_present("#navbar-oops")
 
+    def testEditDescription(self):
+        b = self.browser
+
+        self.createVm("mac", running=False)
+        self.login_and_go("/machines")
+        self.waitPageInit()
+        self.goToVmPage("mac")
+
+        self.performAction("mac", "edit-description")
+
+        # Non-ascii chars, unbalanced quotes, backslash, multiple lines
+        desc1 = '"Döscrü\\ptiän \'\'\' كرة القدم'
+        desc2 = 'Second <b>line</b>'
```

Comment:
I hope this does not get rendered as such :)

Reply:
It gets rendered as text, not as HTML.

![image](https://github.com/user-attachments/assets/c8a72a04-20e0-4b52-ab23-6e1a2ae10eb4)